### PR TITLE
Fix ForEachOnRange detekt issues

### PR DESCRIPTION
**Pull request by DeKaN**
### Contributor checklist
-   [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

-   #126

### CODE REVIEW

**Comment by andrewtavis** [comment]
@angrezichatterbox, can you take a first glance here? :)

File: detekt.yml [hunk 1]
```diff
@@ -60,6 +60,6 @@ empty-blocks:
 
 performance:
     ForEachOnRange:
```
Comment (angrezichatterbox) [inline detekt.yml:62]: This rule could be removed completely from the detekt.yml.

**Review by angrezichatterbox — COMMENTED** [review]
Thanks for your PR @DeKaN. It looks good to me :)  Just a small change is required.

File: detekt.yml (hunk 1 above)
Comment (DeKaN) [reply detekt.yml]: Done

**Review by andrewtavis — APPROVED** [review]
Thanks for the continued support, @DeKaN! Looking forward to the next issue you take on 😊

**Pull request merged**